### PR TITLE
checking in files for proxy to display 3 modules

### PR DESCRIPTION
**Pull request by bshresthap**
checking in files for proxy to display 3 modules. Will work on remaining modules in next PR.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<body>
+  <h1>Amazon</h1>
+  <div id="photos"></div>
+  <div id="CustomerQuestions"></div>
+  <div id="CustomerReviews"></div>
+  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <script src="http://localhost:4002/photos_bundle.js"></script>
+  <script src="http://localhost:4001/customerQuestions_bundle.js"></script>
+  <script src="http://localhost:4006/reviews_bundle.js"></script>
```
Comment (amosNash) [inline public/index.html:19]: I see that you are displaying three services onto the page. I was just wondering if you had trouble displaying my description service or if there was any specific reason for only having these three services?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <script>
+    // ReactDOM.render(
```
Comment (amosNash) [inline public/index.html:21]: You can get rid of the commented out code.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    //   React.createElement(photos),
+    //   document.getElementById('photos')
+    // );
+    ReactDOM.render(
```
Comment (amosNash) [inline public/index.html:25]: I think you can get rid of this code. Each script above should be creating and rendering an element onto the page.

File: server/server.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const morgan = require('morgan');
+const path = require('path');
+const app = express();
+const port = 3000;
+const cors = require('cors');
+app.use(cors());
```
Comment (amosNash) [inline server/server.js:7]: This all looks good!

**Review by amosNash — COMMENTED** [review]
This all looks good to me!

File: public/index.html (hunk 1 above)
Comment (bshresthap) [reply public/index.html:19]: Hi Amos , I was running into error and I might have the old version of the code. It was already late and I had to submit the video, so went ahead with just few. I will be trying it out today for remaining ones after getting latest and will let you know.

File: public/index.html (hunk 2 above)
Comment (bshresthap) [reply public/index.html:21]: Yes, I am planning to do so after discussing few question on this with TM.

File: public/index.html (hunk 3 above)
Comment (bshresthap) [reply public/index.html:25]: Yes, I am planning to do so after discussing few question on this with TM.

**Pull request merged**